### ISSUE TITLE
QuickBooks API v3 Docs

### ISSUE THREAD

**Issue by MattRogish** [ruckus/quickeebooks#39]
Intuit has released their roadmap for the v3 API, rolling out over April-Oct of this year:
http://docs.developer.intuit.com/@api/deki/files/810/QuickBooksAPIv3Webinar-Feb2013.pdf

Might be worth starting the discussion on how we're gonna support v2 and v3 alongside (docs say you can only be v2 or v3 - not both) -- or should it be a new project?

**Comment by ruckus** [comment]
Yeah I saw this webinar back in February. 

When they say v2 and v3 are not supported alongside what they meant is that an app that authorizes against v2 cannot then turn around make v3 calls. The app will have to disconnect and re-auth thru v3.

I do think we should make a new project as the entities and operations will change, and juggling both in one project will just get messy.

From my own perspective I'm not jumping on v3 just because the timeline is so far out in terms of readiness, so I feel like I will cross that bridge when its ready. On top of that Intuit has very little documentation on v3 itself so developing a consumer API without any of the details will likely be futile.

So basically, eventually I'd like to support v3 - just when its actually ready.

**Comment by MattRogish** [comment]
Sounds good to me. :D Thanks!
